Description: Upstream changes introduced in version 0.42-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 sshuttle (0.42-1) unstable; urgency=low
 .
   * Initial release (Closes: 601626)
   * Write manpage for the Debian release
 .
 The person named in the Author field signed this changelog entry.
Author: Javier Fernandez-Sanguino Pen~a <jfs@debian.org>
Bug-Debian: http://bugs.debian.org/601626
Last-Update: 2010-11-17
Forwarded: no

--- apenwarr-sshuttle-7d0d5ce/Makefile	1970-01-01 01:00:00.000000000 +0100
+++ sshuttle/Makefile	2010-11-17 01:56:03.185275208 +0100
@@ -0,0 +1,20 @@
+PANDOC:=$(shell \
+	if pandoc </dev/null 2>/dev/null; then \
+		echo pandoc; \
+	else \
+		echo "Warning: pandoc not installed; can't generate manpages." >&2; \
+		echo '@echo Skipping: pandoc'; \
+	fi)
+
+default: all
+
+
+sshuttle.8: sshuttle.md
+
+%.8: %.md
+	$(PANDOC) -s -r markdown -w man -o $@ $<
+
+all: sshuttle.8
+
+clean:
+	rm -f *~ */*~ .*~ */.*~ *.8 *.tmp */*.tmp *.pyc */*.pyc
